### ISSUE TITLE
Angular 7 a11y implementation no accessibility tests or failures 

### ISSUE THREAD

**Issue by jameshughes7** [storybooks/storybook#5752]
**Describe the bug**
No accessibility tests or failures being produced after including a11y addon to my **Angular project** (followed https://github.com/storybooks/storybook/tree/master/addons/a11y)

**To Reproduce**
1. $ npm install -D @storybook/addon-a11y
2. import '@storybook/addon-a11y/register';
4. imported **storiesOf** and **checkA11y** into stories file
5. npm run storybook

**Expected behavior**
Expected to see Accessibility tests failing or passing

**Screenshots**
<img width="1422" alt="angular_a11y_addon_not_working" src="https://user-images.githubusercontent.com/30200120/53341128-9c821780-3902-11e9-8a8b-7f2f53d5c674.png">

**Code snippets**
**button.component.ts**
```
import { Component, Input, Output, EventEmitter } from "@angular/core";

enum ButtonStyle {
  primary = "primary",
  disabled = "disabled"
}

@Component({
  selector: "button",
  templateUrl: "./button.component.html",
  styleUrls: ["./button.component.scss"]
})
export class ButtonComponent {
  @Input() text: string;
  @Input() disabled: boolean = false;
  @Input() style: ButtonStyle = ButtonStyle.primary;

  @Output() btnClick: EventEmitter<MouseEvent> = new EventEmitter();

  handleClick(event: MouseEvent): void {
    this.btnClick.emit(event);
  }
}
```

**a11y.stories.ts**
```
import { storiesOf } from "@storybook/angular";
import { checkA11y } from "@storybook/addon-a11y";

import { ButtonComponent } from "../projects/component-library/src/lib/buttons/button/button.component";

storiesOf("Buttons/Button", module)
  .addDecorator(checkA11y)

  .add("Accessible", () => ({
    component: ButtonComponent,
    props: {
      text: "Accessible",
      style: "primary"
    }
  }))

  .add("Inaccessible", () => ({
    component: ButtonComponent,
    props: {
      text: "Inaccessible",
      disabled: true
    }
  }));
```

** I copy pasted the code snippet from the Getting Started doc  and got the following error:**
```
Module parse failed: Unterminated regular expression (5:69) You may need an appropriate loader to handle this file type. | storiesOf("Buttons/FresButton", module) | .addDecorator(checkA11y) > .add("Accessible", function () { return Accessible; }, button < /button>) | .add("Inaccessible", function () { return style = {}; }, { backgroundColor: "red", color: "darkRed" }, > | Inaccessible, button
    Error: Module parse failed: Unterminated regular expression (5:69)
You may need an appropriate loader to handle this file type.
| storiesOf("Buttons/FresButton", module)
|     .addDecorator(checkA11y)
>     .add("Accessible", function () { return Accessible; }, button < /button>)
|     .add("Inaccessible", function () { return style = {}; }, { backgroundColor: "red", color: "darkRed" },  >
|     Inaccessible, button
    at Object../stories/a11y.stories.ts (http://localhost:6006/main.d419c1ea8ca764b913b5.bundle.js:186:7)
    at __webpack_require__ (http://localhost:6006/runtime~main.d419c1ea8ca764b913b5.bundle.js:782:30)
    at fn (http://localhost:6006/runtime~main.d419c1ea8ca764b913b5.bundle.js:150:20)
    at webpackContext (http://localhost:6006/main.d419c1ea8ca764b913b5.bundle.js:159:9)
    at http://localhost:6006/main.d419c1ea8ca764b913b5.bundle.js:29:12
    at Array.forEach (<anonymous>)
    at loadStories (http://localhost:6006/main.d419c1ea8ca764b913b5.bundle.js:28:14)
    at ConfigApi._renderMain (http://localhost:6006/vendors~main.d419c1ea8ca764b913b5.bundle.js:72342:20)
    at render (http://localhost:6006/vendors~main.d419c1ea8ca764b913b5.bundle.js:72296:17)
    at http://localhost:6006/vendors~main.d419c1ea8ca764b913b5.bundle.js:72325:9
```

**System:**
 - OS: [MacOS]
 - Browser: [chrome]
 - Framework: [angular]
 - Addons: [a11y]
 - Version: ["@storybook/addon-a11y": "^4.1.13",  "@storybook/angular": "^4.1.12"]

**Additional context**
I'm a relatively newbie to Storybook and Angular so any help here would be greatly appreciated.

**Comment by jameshughes7** [comment]
Please ignore step 3 in **To Reproduce** section.

**Comment by shilman** [comment]
@jameshughes7 Note that `addon-a11y` is currently not working in 4.x but is apparently working in 5.0-rc:

https://github.com/storybooks/storybook/issues/5543

Yours might a different issue though. Please take a look and LMK what you think!

**Comment by jameshughes7** [comment]
HI @shilman thanks for the quick response! I an updating my SB version and trying it out. Will let you know. 👍

**Comment by jameshughes7** [comment]
@shilman That worked very well. I like the new layouts BTW. Thanks for the help!

<img width="828" alt="sb_5 0 0_rc5_a11y_success" src="https://user-images.githubusercontent.com/30200120/53353770-60a87b80-391d-11e9-94e4-86f5b6789aab.png">

**Comment by shilman** [comment]
Awesome possum!
